### PR TITLE
Update multus readme to improve user experience

### PR DESCRIPTION
**Pull request by 12345lcr**
## What this PR does / why we need it

Update multus readme to improve user experience. 

Following this doc:
https://docs.google.com/document/d/1WEZaKEBQ9A5xoOkbX4ZWREDvEw3zCwfu1oIrRWff8Y4/edit#heading=h.bzeiwt4lj7tb

## Which issue(s) this PR fixes

Fixes: #

### CODE REVIEW

**Comment by 12345lcr** [comment]
Hi @LappleApple , I draft the pr to address your suggestions. Feel free to take a look.

File: addons/packages/multus-cni/3.7.1/README.md [hunk 1]
```diff
@@ -1,24 +1,41 @@
...
 
-## Supported Providers
+This documentation provides information about the specific TCE package. Please visit the [TCE package management page](https://tanzucommunityedition.io/docs/v0.11/package-management/) for general information about installation, removal, troubleshooting, and other topics.
 
-The following table shows the providers this package can work with.
+## Installation
 
-| AWS  |  Azure  | vSphere  | Docker |
-|:---:|:---:|:---:|:---:|
-| ✅  |  ✅  | ✅  | ✅ |
```
Comment (stmcginnis) [inline addons/packages/multus-cni/3.7.1/README.md:11]: I believe we wanted to standardize on having this supported provider table in each package's README. We may want to keep this part.

Comment (12345lcr) [reply addons/packages/multus-cni/3.7.1/README.md:11]: Hi @stmcginnis , it was not deleted, but moved to a lower place.

I am following this doc that provided by @LappleApple to adjust the titles' sequence and its contents
https://docs.google.com/document/d/1WEZaKEBQ9A5xoOkbX4ZWREDvEw3zCwfu1oIrRWff8Y4/edit#heading=h.bzeiwt4lj7tb

I should add this doc to the pr description

**Review by wjun — APPROVED** [review]
LGTM

File: addons/packages/multus-cni/3.7.1/README.md (hunk 1 above)
Comment (seemiller) [reply addons/packages/multus-cni/3.7.1/README.md:11]: Chenrui is correct that the structure of the readme is changing.

**Review by seemiller — APPROVED** [review]

**Pull request merged**